cmd/registry.go

```diff
@@ -117,12 +129,25 @@ This dataset will no longer show up in search results.`,
 		},
 	}
 
-	signup.Flags().StringVar(&o.Username, "username", "", "registry username to prove")
-	signup.Flags().StringVar(&o.Password, "password", "", "registry password")
-	signup.Flags().StringVar(&o.Email, "email", "", "email")
+	signup.Flags().StringVar(&o.Username, "username", "", "desired username")
+	signup.Flags().StringVar(&o.Password, "password", "", "a new password for online login")
```

Comment:
So, this isn't urgent, and it's already the status quo, but I would really like at some point in the future to move away from requiring users to enter passwords as command-line flags. It's bad for multiple reasons. My preference would be to always use something like https://godoc.org/golang.org/x/crypto/ssh/terminal#Terminal.ReadPassword which won't echo back to the terminal, and provides just a bit more protection for sensitive info.

Reply:
love it, mind filing an issue?